src/pages/whats-happening/releases/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Releases
+description:
+  Learn about our latest updates, our release history, and what the team has
+  planned for Carbon’s future.
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListRow,
+  StructuredListCell,
+  StructuredListBody,
+  Link,
+  UnorderedList,
+} from 'carbon-components-react';
+
+<PageDescription>
+
+Learn about our latest updates, our release history, and what the team has
+planned for Carbon’s future.
+
+</PageDescription>
+
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Changelog"
+    href="https://github.com/carbon-design-system/carbon/releases">
+    <MdxIcon name="github" />
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Roadmap"
+    href="https://app.zenhub.com/workspaces/system-squad-593830641344b813db10934d/roadmap">
+
+![ZenHub icon](images/z-icon.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Releases
+
+Each Carbon Design System release includes package releases that are
+independently versioned following the industry-standard
+[semantic versioning](https://semver.org) specification.
+
+The Carbon team also ships package minor releases every other week and when
+necessary, patch releases between minor versions to address bugs. Read more in
+Carbon's
+[release radar](https://github.com/carbon-design-system/carbon/wiki/Release-radar).
+
+### Carbon v11
+
+Carbon v11 focuses on quality of life updates for designers and developers
+including token and prop renaming for ease of use, component API updates for a
+more predictable developer experience, and introducing CSS grid for accurate
+layout building in line with the design language.
+
+With no changes to the IBM Design Language, v11 will not require any
+brand-driven product redesigns and teams are free to update at their
+convenience.
+
+### Carbon v10
+
+Carbon v10 implemented our newly-updated brand expression (IBM Design Language
+v2) into our components. The updated components required product teams to
+redesign product UIs and begin a migration plan to comply with IBM brand
+experience standards.
+
+## Release history
+
+<!-- prettier-ignore-start -->
+| Carbon Design System | IBM Design Language | Carbon Elements | Carbon Components | Carbon React |
+| -------------------- | ------------------- | --------------- | ----------------- | ------------ |
+| <div>Carbon v11</div><div>— Q2 2021</div><div>— Improves token and prop naming</div><div>— Adds light/dark mode support</div><div>— Uses CSS grid</div> | [IDL v2](https://www.ibm.com/design/language) | v11.x | v11.x | v8.x |
```

Comment:
Is there a reason for the wrapper divs here? If you need to target them with css, a span with a classname would be more appropriate.

Reply:
@vpicone I need line breaks. Ideally, unordered list elements, but those render as 16px font size which is bigger than the markdown table font size, so that's not an option right now. I previously had `<br />` elements in there, but Prettier formatting was busting the table. Then I switched it to divs and Prettier still busted the table. Then I added the Prettier ignore statements and called it good.

Reply:
Right on. If you're ignoring it anyway, can we go back to break elements to get the line breaks?

I just don't want the next person editing the table to be confused with the empty wrappers.

Reply:
@vpicone will do, I'll swap back to line break elements.